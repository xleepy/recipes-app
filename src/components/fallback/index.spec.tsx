import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { Fallback } from '.';

describe('Fallback tests', () => {
  it('should render fallback with error', () => {
    const { asFragment } = render(<Fallback error={new Error('test')} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
