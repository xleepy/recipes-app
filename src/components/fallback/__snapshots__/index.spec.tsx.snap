// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Fallback tests > should render fallback with error 1`] = `
<DocumentFragment>
  <div
    role="alert"
  >
    <p>
      Something went wrong:
    </p>
    <pre
      style="color: red;"
    >
      test
    </pre>
  </div>
</DocumentFragment>
`;
